modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/disruptor/inbound/InboundQueueEvent.java

```diff
@@ -159,6 +159,11 @@ public String eventInfo() {
         return eventType.toString();
     }
 
+    @Override
+    public boolean isActionableWhenPassive() {
```

Comment:
What will happen when a subcription comes binding to a new queue name? The queue event will be taken in and the queue will get created, and the binding will not be taken in. Will that be handled properly when the subscription tries to get subscribed from the other node?

Reply:
Can you suggest a test scenario to verify this?